splink/dialects.py

```diff
@@ -40,16 +50,26 @@ def name(self):
     def levenshtein_function_name(self):
         return "levenshtein"
 
+    @property
+    def jaro_winkler_function_name(self):
+        return "jaro_winkler"
+
 
 class SqliteDialect(SplinkDialect):
     @property
     def name(self):
         return "sqlite"
 
+    # SQLite does not natively support string distance functions.
+    # Instead, these operate as UDF sourced from 'fuzzyfinder'.
```

Comment:
```suggestion
    # However, python UDFs are registered automatically by Splink
```

I've tried to phrase the above in a way that isn't tied to the specific implementation in case it changes, but alerts future readers to the potential confusion (i.e. that jaro-winkler doesn't exist in sqlite)